js/client-mainmenu.js

```diff
@@ -1240,17 +1240,19 @@
 		},
 		selectFolder: function (key) {
 			var keyExists = false;
-			for (var i = this.folderNotExpanded.length - 1; i > -1; i--) {
-				if (this.folderNotExpanded[i] === key) {
+			var folderNotExpandedTemp = this.folderNotExpanded.filter(function (folder) {
```

Comment:
I though we went over `temp` being a bad name? Please just call this `folderNotExpanded`.